### PR TITLE
Added highlight for selected line in checkbox

### LINKED ISSUE #479: Highlight current selection
Hi!
Is it possible to highlight current line completely? Perhaps, change text color or something like that?

   o Long-long-long-long-long-long-long-long-long-long string 1
   o Long-long-long-long-long-long-long-long-long-long string 2
   o Long-long-long-long-long-long-long-long-long-long string 3
\> o **Long-long-long-long-long-long-long-long-long-long string 4**
   o Long-long-long-long-long-long-long-long-long-long string 5

Btw, don't know how to add spaces in the beginning of a line in markdown, but it seems that some indent would be also good for that purpose :-)

### PR DESCRIPTION
**Pull request by aliaksandr-yermalayeu**
fixes #479 

There is inconsistency between list and checkbox selected line. It is highlighted in list and ignored in checkbox module. It is hard to follow what item is currently selected, when there are to many similar items with checkboxes. This fix changes code in order to highlight selected line with cyan color.

### CODE REVIEW

File: packages/inquirer/lib/prompts/checkbox.js [hunk 1]
```diff
@@ -226,9 +226,12 @@ function renderChoices(choices, pointer) {
       output += ' - ' + choice.name;
       output += ' (' + (_.isString(choice.disabled) ? choice.disabled : 'Disabled') + ')';
     } else {
-      var isSelected = i - separatorOffset === pointer;
-      output += isSelected ? chalk.cyan(figures.pointer) : ' ';
-      output += getCheckbox(choice.checked) + ' ' + choice.name;
+      var line = getCheckbox(choice.checked) + choice.name;
```
Comment (SBoudrias) [inline packages/inquirer/lib/prompts/checkbox.js]: When rewriting this line, the blank space between the checkbox and the option name has been removed (e.g. `[]Choice A` vs `[] Choice A`)

Comment (aliaksandr-yermalayeu) [reply packages/inquirer/lib/prompts/checkbox.js]: My mistake. Blank space was restored in the latest commit.

**Comment by SBoudrias** [comment]
Thanks!

**Comment by sergei-startsev** [comment]
@SBoudrias is there a chance to publish the changes?

**Comment by SBoudrias** [comment]
@sergei-startsev I wasn't personally in a hurry, but as you requested it I'll cut a release tonight.

**Comment by sergei-startsev** [comment]
@SBoudrias thanks for the release!
